src/python/zigopt/optimize/sources/base.py

```diff
@@ -83,6 +83,60 @@ def extract_open_suggestion_datas(optimization_args):
       open_suggestion_datas = [s.suggestion_meta.suggestion_data for s in optimization_args.open_suggestions]
     return open_suggestion_datas
 
+  def _get_dimension_index(self, hyper_opt_dimension):
+    # pylint: disable=too-many-return-statements
+    if hyper_opt_dimension < 11:
```

Comment:
odd nit: can we do `<= 10`, etc? feel like seeing 10,20,25,30 is more parseable than `11, 21, 26, 31`

Reply:
yeah that's fine by me